tests/test_models/test_engine/test_db_storage.py

```diff
@@ -70,19 +70,24 @@ def test_dbs_func_docstrings(self):
 
 class TestFileStorage(unittest.TestCase):
     """Test the FileStorage class"""
-    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    '''@unittest.skipIf(models.storage_t != 'db', "not testing db storage")'''
```

Comment:
why do you comment this line?

Reply:
The tests don't work with these lines actived. I have to comment to get the proves right. It doesn't affect the results.